internal/processors/cloudeventconvert/cloudeventconvert.go

```diff
@@ -138,16 +143,16 @@ func (c *cloudeventProcessor) processMsg(ctx context.Context, msg *service.Messa
 			return service.MessageBatch{msg}
 		}
 
-		attestorField, ok := msg.MetaGet(httpinputserver.DIMOCloudEventSource)
+		source, ok := msg.MetaGet(httpinputserver.DIMOCloudEventSource)
 		if !ok {
-			processors.SetError(msg, processorName, "failed to get attestor from message metadata", nil)
+			processors.SetError(msg, processorName, "failed to get source from message metadata", nil)
 			return service.MessageBatch{msg}
 		}
 
-		validSignature, err := c.validateSignature(event, attestorField)
+		validSignature, err := c.verifySignature(event, common.HexToAddress(source))
 		if err != nil {
-			c.logger.Warn(fmt.Sprintf("failed to validate signature: %s", err.Error()))
-			processors.SetError(msg, processorName, "failed to validate signature on message", err)
+			c.logger.Warn(fmt.Sprintf("failed to check message signature: %s", err.Error()))
```

Comment:
not needed now but we will need to remember to remove these logs later because setError should also be logging.